app/_hub/kong-inc/rate-limiting/index.md

```diff
@@ -210,79 +210,71 @@ X-RateLimit-Limit-Minute: 10
+In this scenario, because accuracy is important, the `local` policy is not an option. Consider the support effort you might need 
+for Redis, and then choose either `cluster` or `redis`. 
+
+You could start with the `cluster` policy, and move to `redis`
+if performance reduces drastically. 
 
-The recommendation is to start with the `cluster` policy, with the option to move over to `redis`
-if performance reduces drastically. Keep in mind existing usage metrics cannot
-be ported from the datastore to redis. Generally with shortlived metrics (per second or per minute)
-this is not an issue, but with longer lived ones (months) it might be, so you might want to plan
-your switch more carefully.
+Do remember that you cannot port the existing usage metrics from the datastore to Redis. 
+This might not be a problem with shortlived metrics (for example, seconds or minutes)
+but if you use metrics with a longer time frame (for example, months), plan
+your switch carefully.
 
 ### Backend protection
 
-If accuracy is of lesser importance, the `local` policy can be used. It might require some experimenting
-to get the proper setting. For example, if the user is bound to 100 requests per second, and you have an
-equally balanced 5 node Kong cluster, setting the `local` limit to something like 30 requests per second
-should work. If you are worried about too many false-negatives, increase the value.
+If accuracy is of lesser importance, choose the `local` policy. You might need to experiment a little
+before you get a setting that works for your scenario. As the cluster scales to more nodes, more user requests are handled. 
+When the cluster scales down, the probability of false negatives increases. So, adjust your limits when scaling.
 
-Keep in mind as the cluster scales to more nodes, the users will get more requests granted, and likewise
-when the cluster scales down the probability of false-negatives increases. So in general, update your
-limits when scaling.
+For example, if a user can make 100 requests every second, and you have an
+equally balanced 5-node Kong cluster, setting the `local` limit to something like 30 requests every second
+should work. If you see too many false negatives, increase the limit.
 
-The above-mentioned inaccuracy can be mitigated by using a consistent-hashing load balancer in front of
-Kong, which ensures the same user is always directed to the same Kong node. This will both reduce the
-inaccuracy and prevent the scaling issues.
+To minimise inaccuracies, consider using a consistent-hashing load balancer in front of
+Kong. The load balancer ensures that a user is always directed to the same Kong node, thus reducing
+inaccuracies and preventing scaling problems.
 
-Most likely the user will be granted more than was agreed when using the `local` policy, but it will
-effectively block any attacks while maintaining the best performance.
+It might happen that users can make more requests than their quota, but the `local` policy
```

Comment:
Can you please remove all of this text? `It might happen that users can make more requests than their quota, but the `local` policy can effectively block any attacks while maintaining good performance.` We've determined that this info is not necessary.